### PR TITLE
Add constraints for pydantic

### PR DESCRIPTION
**Pull request by jaimergp**

### CODE REVIEW

**Comment by jaimergp** [comment]
@Czaki - just to make sure, did you want to pin `pydantic<2` or `pydantic<=2.0` here?

**Review by Czaki — APPROVED** [review]

File: napari_plugin_manager/qt_package_installer.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ def constraints() -> Sequence[str]:
         """
         Version constraints to limit unwanted changes in installation.
         """
-        return [f"napari=={_napari_version}"]
+        return [f"napari=={_napari_version}", "pydantic<=2.0"]
```
Comment (Czaki) [inline napari_plugin_manager/qt_package_installer.py:88]:
```suggestion
        return [f"napari=={_napari_version}", "pydantic<2.0"]
```

File: napari_plugin_manager/qt_package_installer.py [hunk 2]
```diff
@@ -230,7 +230,7 @@ def constraints() -> Sequence[str]:
         pin_level = 2 if is_dev else 3
         version = ".".join([str(x) for x in _napari_version_tuple[:pin_level]])
 
-        return [f"napari={version}"]
+        return [f"napari={version}", "pydantic<=2.0"]
```
Comment (Czaki) [inline napari_plugin_manager/qt_package_installer.py:233]:
```suggestion
        return [f"napari={version}", "pydantic<2.0"]
```

**Comment by Czaki** [comment]
Why I put `<=` ...

File: napari_plugin_manager/qt_package_installer.py (hunk 2 above)
Comment (jaimergp) [reply napari_plugin_manager/qt_package_installer.py:233]: Then https://github.com/napari/napari/pull/5785 needs to be amended too.

**Pull request merged**